bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -301,17 +226,7 @@ <h2 class="primary go">{{ _('Mozilla Developer Network') }}</h2>
             </div>
           </a>
         </li>
-      {% elif l10n_has_tag('firefox_friends') %}
-        <li id="promo-11" class="item promo-large-landscape firefox-friends" tabindex="0">
-          <h2 class="primary go">{{ _('Introducing <br>Firefox Friends') }}</h2>
-          <a class="panel-link" href="https://friends.mozilla.org" data-promotion-id="promo-11" data-promotion-name="Introducing Firefox Friends" data-promotion-class="promo-large-landscape" data-promotion-type="tile">
-            <div class="secondary">
-              <h3>{{ _('Join. Share. <br>Get rewarded.') }}</h3>
-              <p class="more">{{ _('Join Firefox Friends') }}</p>
-            </div>
-          </a>
-        </li>
-      {% else %}
+        {# Since MDN promo is evergreen we can't use a l10n tag so comment out the fallback.
```

Comment:
@flodolo - just checking that strings within this comment are still extracted ok? (I wasn't 100% sure if this is ok)